extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -38,13 +40,49 @@ export const buildOpenAttachmentElement = (element) =>
     </a>`;
 
 /**
- * Renders the open attachment UI affordance.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+export const buildOpenInlineAttachmentElement = (element) =>
+  htmlFor(element)`
+    <a class="
+        i-amphtml-story-page-open-attachment i-amphtml-story-system-reset"
+        role="button">
+      <div class="i-amphtml-story-inline-page-attachment-chip">
+        <div role="img" class="i-amphtml-story-inline-page-attachment-img"></div>
+        <div role="img" aria-label="Upward pointing arrow to indicate scroll up to view page attachment" class="i-amphtml-story-inline-page-attachment-arrow"></div>
```

Comment:
The aria-label needs to be on the parent `role=button` element actually. 
My bad, I'm realizing this after testing it on a device. We definitely should test a11y on a device while developing for a11y.

We're removing the label here but general feedback about it:
If it's a UI element, it should describe the action, not the image. This one could be something like "Open attachment" since that's the action it describes. It's also good to keep them simple because they need to be translated for i18n.